js/backbone.upload-manager.js

```diff
@@ -69,6 +69,13 @@
             // Add local events handlers
             this.bindLocal();
         },
+        // Max Wong: Cannot add same file after cancelled #21
```

Comment:
Git blame will give you the authorship, no worries you can remove these comments :wink: